packages/react/src/Link/Link.test.tsx

```diff
@@ -15,26 +15,6 @@ describe('Link', () => {
     expect(link).toHaveAttribute('href', '#')
   })
 
-  it('renders standalone variant', () => {
```

Comment:
We're missing the 'renders a design system BEM class name' test now

Reply:
Could you add it so we can merge the PR?

Reply:
Done. I kind of updated all the Link tests 😁 Can you check?